### DISCUSSION TITLE
小型語言模型使遊戲角色對話更生動

### DISCUSSION THREAD

**Discussion by Raiden1121** [1101-GS4719-C/Python-Community-GS4719-C#2065] [General]
https://www.cna.com.tw/news/ait/202408210241.aspx

看完這篇報導後讓我馬上聯想到一部電影「脫稿玩家」，這種生成式AI的NPC不僅使遊戲不再單一，還能協助玩家更好的客製化遊戲。然而，生成式AI也勢必會影響製作遊戲的難度（如bug等......），因此，要達成生成式AI的遊戲仍需要工程師共同的努力。

此外，我認為生成式AI可以在未來自動化調整玩家遊玩遊戲的難度，又或者是隨著遊戲的過程，自動生成出不同的結局，使玩家能夠感受不同的體驗。
你認為生成式AI將如何影響未來的遊戲產業？會產生出什麼樣的遊戲？

**Comment by hunteryang0108** [comment]
將小型語言模型嵌入遊戲中確實是一個不錯的想法，現在已經有RPG遊戲開發公司正在這樣做了。有別與以往單調的文本劇情，以及被遊戲編劇所設定好的固定套路，透過小型語言模型的幫助，玩家能體驗到更貼近自身想法的劇情，遊戲公司也能減少編寫有趣劇情所需投入的成本。不過小型語言模型的創意仍然有限，能否提供出令玩家感興趣的內容仍需要觀察。
